docs/usage.rst

```diff
@@ -172,9 +172,32 @@ container based on the given image name. This assumes the ``fv3config`` package
 ``fv3gfs`` python wrapper are installed inside the container, along with any
 dependencies.
 
+
 The python interface is very similar to the command-line interface, but is split into
 separate functions based on where the model is being run.
 
+Customizing the model execution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``runfile`` is the python script that will be executed by mpi, which
+typically imports the ``fv3gfs`` module, and then performs some time stepping.
+The default behavior is to use a pre-packaged runfile ``-m fv3config.fv3run``
```

Comment:
Remove the ``-m fv3config.fv3run`` because it may confuse users about what they're supposed to give (a filename, not a module like we use when doing the default). You can just say "pre-packaged runfile".

Reply:
done.